client/my-sites/checkout/checkout-thank-you/header.jsx

```diff
@@ -238,6 +240,18 @@ export class CheckoutThankYouHeader extends PureComponent {
 		);
 	}
 
+	visitDomain = ( event ) => {
+		event.preventDefault();
+
+		const { primaryPurchase, selectedSite } = this.props;
+
+		this.props.recordTracksEvent( 'calypso_thank_you_view_site', {
+			product: primaryPurchase.productName,
```

Comment:
maybe we should add a prop here to distinguish the single domain vs. multiple domains purchase?

Reply:
Something like this?